CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.0.0
+
+- [Breaking change] Min Flutter SDK >=3.0.0
+- Add validate international number without region code
```

Comment:
Can you please add this? This was also a change that needs to be mention

- Separate `regionCode` and `countryCode`